charts/quickwit/values.yaml

```diff
@@ -313,50 +348,33 @@ bootstrap:
 
 
 # Quickwit configuration
+# Warning: This config is writed directly into a configMap
+# to avoid passing sensitive value you can pass environment variables.
+# https://quickwit.io/docs/configuration/node-config#using-environment-variables-in-the-configuration
 config:
-  # Metastore configuration.
-  # Default to `qwdata/indexes` if not set.
-  # S3 metastore
-  # metastore_uri: s3://quickwit/indexes
-  #
-  # Postgres metastore
+  version: 0.8
+  listen_address: 0.0.0.0
+  gossip_listen_port: 7282
+  data_dir: /quickwit/qwdata
+  default_index_root_uri: s3://quickwit/indexes
+
   # postgres:
-  #   host: ""
-  #   port: 5432
-  #   database: metastore
-  #   username: quickwit
-  #   password: ""
   #   max_num_connections: 50
 
-  # Storage configuration.
   # storage:
     # s3:
       # endpoint: "http://custom-s3-endpoint"
       # region: eu-east-1
       # We recommend using IAM roles and permissions to access Amazon S3 resources,
       # but you can specify a pair of access and secret keys if necessary.
       # access_key_id: <my access key>
-      # secret_access_key: <my secret key>
+      # secret_access_key: ${AWS_ACCESS_KEY_ID}
     # azure:
-      # account: "my-azure-blob-account-name"
-      # access_key: "my-azure-blob-access-key"
-
-  default_index_root_uri: s3://quickwit/indexes
-
-  # Indexer settings
-  # indexer:
-  #   split_store_max_num_bytes: 200G
-  #   split_store_max_num_splits: 10000
-  # Ingest API settings
-  # ingest_api:
-  #   max_queue_memory_usage: 2GiB
-  #   max_queue_disk_usage: 4GiB
-  # Searcher settings
-  # searcher:
-  #   fast_field_cache_capacity: 10G
-  #   split_footer_cache_capacity: 1G
-  #   max_num_concurrent_split_streams: 100
+      # account: "<my account name>"
+      # access_key: ${QW_AZURE_STORAGE_ACCESS_KEY}
 
+# Seed configuration
+seed:
```

Comment:
@idrissneumann have you tested seeding as well?

Reply:
Hi.

Now yes, this is the overrided value file:

```yaml
---
bootstrap:
  enabled: true

seed:
  indexes:
    - version: 0.8
      index_id: my-index
      doc_mapping:
        field_mappings:
          - name: timestamp
            type: datetime
            fast: true
            input_formats:
              - unix_timestamp
            output_format: unix_timestamp_secs
          - name: body
            type: text
        timestamp_field: timestamp
      search_settings:
        default_search_fields: [body]
      indexing_settings:
        merge_policy:
          type: limit_merge
          max_merge_ops: 3
          merge_factor: 10
          max_merge_factor: 12

config:
  default_index_root_uri: s3://qw-indexes
  storage:
    s3:
      endpoint: https://s3.fr-par.scw.cloud
      region: fr-par
      access_key_id: SCWXXXXXX
      secret_access_key: XXXXXXXXXX

environment:
  QW_METASTORE_URI: s3://qw-indexes

# Testing purpose
my_secret: bar

indexer:
  extraEnvFrom:
    - secretRef:
        name: my-secret
```

The result:

```shell
$ kubectl create ns quickwit && helm template . --values values.yaml --values values2.yaml --namespace quickwit | kubectl -n quickwit apply -f - > /dev/null 2>&1
$ kubectl -n quickwit get pods
NAME                                                 READY   STATUS      RESTARTS      AGE
release-name-quickwit-control-plane-c65c78d9-b279k   1/1     Running     1 (27s ago)   117s
release-name-quickwit-index-my-index-6t2np           0/1     Completed   0             24s
release-name-quickwit-index-my-index-bqdt4           0/1     Error       0             105s
release-name-quickwit-index-my-index-fcgd9           0/1     Error       0             74s
release-name-quickwit-index-my-index-wzkh4           0/1     Error       0             117s
release-name-quickwit-indexer-0                      1/1     Running     1 (27s ago)   117s
release-name-quickwit-janitor-d9b7cd749-sv4w7        1/1     Running     1 (27s ago)   117s
release-name-quickwit-metastore-755b8849bc-bnxlw     1/1     Running     0             117s
release-name-quickwit-searcher-0                     1/1     Running     1 (27s ago)   117s
release-name-quickwit-searcher-1                     1/1     Running     0             21s
release-name-quickwit-searcher-2                     1/1     Running     0             15s
$ kubectl -n quickwit port-forward pod/release-name-quickwit-searcher-0 7280:7280
Forwarding from 127.0.0.1:7280 -> 7280
Forwarding from [::1]:7280 -> 7280
```

And we can see that the index has been created:

![Screenshot 2024-06-20 at 10 18 00](https://github.com/quickwit-oss/helm-charts/assets/12884347/b09bcce1-802f-4c80-9fbd-8f810a2e03ca)